config/settings.js

```diff
@@ -27,11 +35,19 @@
  */
 
 window.spinnakerSettings = {
+  defaultProviders: ['aws', 'gce', 'azure', 'cf', 'kubernetes'],
```

Comment:
That would be nice - not sure how difficult it would be, but definitely would be nice.